contracts/rateproducer/rateproducer.json

```diff
@@ -1 +0,0 @@
-"{\"costaricaeos\":{\"transparency\":10,\"testnets\":8,\"tooling\":7,\"infra\":6,\"community\":10},\"alohaeos\":{\"transparency\":10,\"testnets\":8,\"tooling\":7,\"infra\":6,\"community\":10}}"
```

Comment:
@kriskoin  lets not delete this file

Reply:
@xavier506  so sorry it was a mistake, fixed !!